### DISCUSSION TITLE
What is the principle of TDOA2 ANCHRO time synchronization, is it necessary to use wired synchronization.

### DISCUSSION THREAD

**Discussion by lalalalang999** [bitcraze/discussions#318] [Q&A]
There is one  question about the time synchronization method of TDOA2 ANCHOR. According to my understanding of the code, TDAO2 ANCHOR time synchronization actually uses the principle of two-way communication between tags and ANCHOR in a similar TWR method to eliminate the time difference. Is this my understanding correct? The time synchronization of the TDOA2 base station is not necessary to do the wired time synchronization through the clock synchronization pin of the DW1000 chip.

Is there any introduction to the relevant materials and principles of TDOA2 time synchronization? If wired synchronization is not used, can the distance accuracy of TDOA be guaranteed?

**Comment by krichardsson** [comment]
Yes, we are basically doing TWR between the anchors and as a "side effect" we get the TDoA for the tag. We have not tried to use the clock synchronization pin

There is some more information in the [node repo](https://www.bitcraze.io/documentation/repository/lps-node-firmware/master/).
